requirements-dev.in

```diff
@@ -16,4 +16,4 @@ snapshottest
 types-python-dateutil
 werkzeug
 
-pip-tools
+pip-tools
```

Comment:
Also change out of scope

Reply:
Not sure why this is popping up. There isn't a real change to this file